cfgov/v1/wagtail_hooks.py

```diff
@@ -43,6 +43,17 @@ def share_the_page(request, page):
         flush_akamai(page)
 
 
+@hooks.register('after_delete_page')
+def log_page_deletion(request, page):
+    logger.warning(
```

Comment:
Debug or higher would go to the console, and this is a warning, so it should end up in our logs... we save the console output somewhere, right?